### DISCUSSION TITLE
Documentation of v3

### DISCUSSION THREAD

**Discussion by NicolasMahe** [hayes/pothos#1282] [General]
Hi,

I'm looking for the documentation website of the v3. Is it still available somewhere?

Thanks!

**Comment by hayes** [comment]
I don't think it's currently available anywhere.  The API changes are relatively minor though, outside.of the options used when setting up the builder, so 95% of the 4.0 docs should be exactly the same.

**Comment by NicolasMahe** [comment]
Thanks for the answer.

Thanksfully, the doc v3 can be found using the Wayback Machine: https://web.archive.org/web/20240606192759/https://pothos-graphql.dev/
